backends/gstreamer/render-unix/lib.rs

```diff
@@ -6,7 +6,6 @@
 //! wrapping the *appsink* from the Player. And the shared frames are
 //! mapped as texture IDs.
 
-#[macro_use]
 extern crate gstreamer as gst;
```

Comment:
You could remove all the `extern crate` lines everywhere and for renaming use `gst = { package = "gstreamer", ...}` inside the `Cargo.toml`